tsconfig.json

```diff
@@ -26,7 +26,7 @@
 
     /* Modules */
     "module": "commonjs" /* Specify what module code is generated. */,
-    "rootDir": "src" /* Specify the root folder within your source files. */,
+    // "rootDir": "src" /* Specify the root folder within your source files. */,
```

Comment:
Does we need to transpile some TS files outside `src/`?

Reply:
No, we don't. But I had to change it to make the doc generator work (typedoc). I read the docs od the library, but none of the configurations to exclude the patters worked.